oidc/identity.go

```diff
@@ -42,3 +43,15 @@ func IdentityFromClaims(claims jose.Claims) (*Identity, error) {
 
 	return &ident, nil
 }
+
+// CopyAdditonalClaims copies all additional claims to claims
+func (ident *Identity) CopyAdditonalClaims(claims jose.Claims) {
```

Comment:
If you have the point of view that `AdditionalClaims` can or should contain all Claims not in the struct, then I agree.  I was attempting to have it contain only Claims that dex (and others) were unlikely to use in an attempt to avoid any accidental interference.

 https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
 https://tools.ietf.org/html/rfc7519#section-4.1
 https://openid.net/specs/openid-connect-core-1_0.html#IDToken

I will update the PR such that AdditionalClaims will contain all claims not otherwise included in the struct.